IanGRap/Squish
# Squish detector fix

Contributors: Austin

Goal:
- Fix the duuuuuumb bug where players sometimes die when they aren't supposed to

What changed:
- Adjusted the position of the squish colliders to make them more responsive
- Changed the way bottomBlockIsNotFalling so that it uses the BlockAdjacencyQueries rather than the Rigidbody, because it is more reliable.

How to Test:
- Load up Sprint 5 scene and play for a little while. 
    - See if you get random deaths
